packages/augur-ui/src/modules/events/actions/add-update-transaction.ts

```diff
@@ -243,15 +243,15 @@ export const addUpdateTransaction = (txStatus: Events.TXStatus) => async (
         break;
       }
       case CANCELORDER: {
-        const orderId = transaction.params[TX_ORDER_ID];
+        const orderId = transaction.params && transaction.params.order['orderId'];
```

Comment:
change TX_ORDER_ID to be `orderId` and use the constant.  doesn't look like it's used anywhere else

Reply:
updated